### PR TITLE
GUI-1486: ELB Certificate Management

### PR DESCRIPTION
**Pull request by kamalgill**
Implements https://eucalyptus.atlassian.net/browse/GUI-1486

### CODE REVIEW

**Comment by kamalgill** [comment]
Hal, retest this please

**Comment by kamalgill** [comment]
Hal, retest this please

**Comment by kamalgill** [comment]
Hal, retest this please

**Comment by kamalgill** [comment]
Hal, retest this please

**Comment by aliceh** [comment]
@kamalgill  I am updating the cloud, please hold off with retesting

**Comment by kamalgill** [comment]
Hal, retest this please

**Comment by aliceh** [comment]
Good to merge, test failed due to random failure in Selenium RC.

File: eucaconsole/static/sass/eucaconsole.scss [hunk 1]
```diff
@@ -882,6 +892,9 @@ textarea.hidden {
 
 .reveal-modal {
     z-index: 9999995;
+    [data-tooltip] {
+        z-index: 99999999;
+    }
```
Comment (dkavanagh) [inline eucaconsole/static/sass/eucaconsole.scss]: Was this to get tooltips showing when in a modal? I have that problem in GUI-1587 and stack wizard params tab (though not in modal there)

File: eucaconsole/static/js/pages/elb_view.js [hunk 2]
```diff
@@ -0,0 +1,381 @@
+/**
```
Comment (dkavanagh) [inline eucaconsole/static/js/pages/elb_view.js]: Pretty hard to review diffs when filename also changes!

File: eucaconsole/static/js/pages/eucaconsole_utils.js [hunk 3]
```diff
@@ -82,6 +82,22 @@ angular.module('EucaConsoleUtils', ['CustomFilters', 'ngSanitize'])
...
+                selector = tipBubble.attr('data-selector');
+            $('#' + selector).show().css({'z-index': '9999999999'});
+        });
+        $(document).on('mouseout', tooltipSelector, function () {
+            var tipBubble = $(this),
+                selector = tipBubble.attr('data-selector');
+            $('#' + selector).hide();
+        });
+    };
+})
```
Comment (dkavanagh) [inline eucaconsole/static/js/pages/eucaconsole_utils.js]: ah, interesting. Once this is merged, I'll have to check out some problem tooltips I had.

File: eucaconsole/templates/dialogs/select_certificate_dialog.pt [hunk 4]
```diff
@@ -46,28 +49,23 @@
...
+                            Add Certificate
+                        </span>
                     </div>
                 </div>
             </form>
             <a href="#" id="select-certificate-close-link-form-backend" class="close-reveal-modal">&#215;</a>
         </div>
         <div ng-show="certificateTab === 'SSL'">
-            <p>
+            <p i18n:translate="">
```
Comment (dkavanagh) [inline eucaconsole/templates/dialogs/select_certificate_dialog.pt]: I thought we couldn't translate entire contents of element when element has children? I've been avoiding that due to previous comment about it.

Comment (kamalgill) [inline eucaconsole/templates/dialogs/select_certificate_dialog.pt]: @dkavanagh: Yep, good catch. Will remove the attribute.

**Comment by dkavanagh** [comment]
2 CI failures are bogus. I'm merging.

**Pull request merged**